src/main/java/com/redhat/cloud/policies/engine/process/EventProcessor.java

```diff
@@ -59,7 +64,12 @@ public void process(Event event) {
         } else {
             LOGGER.debugf("Found %d enabled policies for account %s", enabledPolicies.size(), event.getAccountId());
 
-            PoliciesAction policiesAction = new PoliciesAction();
+            PoliciesAction policiesAction = new PoliciesAction(orgIdConfig.isUseOrgId());
+
+            if (orgIdConfig.isUseOrgId()) {
+                policiesAction.setOrgId(event.getOrgId());
```

Comment:
Same thing here, setting the `orgId` all the time will never cause an issue even if it's missing. It doesn't need to be enabled from configuration.